tests/neural_compressor/test_optimization.py

```diff
@@ -219,6 +219,46 @@ def preprocess_function(examples, tokenizer):
 
             self.assertTrue(torch.equal(model_outputs["logits"], transformers_outputs["logits"]))
 
+    def test_ipex_static_quantization_with_smoothquant(self):
```

Comment:
The only difference with the `test_ipex_static_quantization` is the configuration, could we merge two in one ?

Reply:
Yes, I only kept one test case now. thanks!